examples/vector_databases/redis/getting-started-with-redis-and-openai.ipynb

```diff
@@ -0,0 +1,809 @@
+      "4. Modernist literature (Score: 0.847)\n",
+      "5. Art film (Score: 0.843)\n",
+      "6. Art (Score: 0.842)\n",
+      "7. Byzantine art (Score: 0.841)\n",
+      "8. Postmodernism (Score: 0.84)\n",
+      "9. Cubism (Score: 0.839)\n"
+     ]
+    }
+   ],
+   "source": [
+    "results = search_redis(redis_client, 'modern art in Europe', index_name=HNSW_INDEX_NAME, k=10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 107,
+   "id": "cb799e69",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      " ----- Flat Index ----- \n",
+      "0. Museum of Modern Art (Score: 0.875)\n",
+      "1. Western Europe (Score: 0.868)\n",
+      "2. Renaissance art (Score: 0.864)\n",
+      "3. Pop art (Score: 0.86)\n",
+      "4. Northern Europe (Score: 0.855)\n",
+      "5. Hellenistic art (Score: 0.853)\n",
+      "6. Modernist literature (Score: 0.847)\n",
+      "7. Art film (Score: 0.843)\n",
+      "8. Central Europe (Score: 0.843)\n",
+      "9. European (Score: 0.841)\n",
+      "Flat index query time: 0.19 seconds\n",
+      "\n",
+      " ----- HNSW Index ------ \n",
+      "0. Museum of Modern Art (Score: 0.875)\n",
+      "1. Renaissance art (Score: 0.864)\n",
+      "2. Pop art (Score: 0.861)\n",
+      "3. Hellenistic art (Score: 0.853)\n",
+      "4. Modernist literature (Score: 0.847)\n",
+      "5. Art film (Score: 0.843)\n",
+      "6. Art (Score: 0.842)\n",
+      "7. Byzantine art (Score: 0.841)\n",
+      "8. Postmodernism (Score: 0.84)\n",
+      "9. Cubism (Score: 0.839)\n",
+      "HNSW index query time: 0.134 seconds\n",
+      " ------------------------ \n"
+     ]
+    }
+   ],
+   "source": [
+    "# compare the results of the HNSW index to the FLAT index and time both queries\n",
+    "def time_queries(iterations: int = 10):\n",
+    "    print(\" ----- Flat Index ----- \")\n",
+    "    t0 = time.time()\n",
+    "    for i in range(iterations):\n",
+    "        results_flat = search_redis(redis_client, 'modern art in Europe', k=10, print_results=False)\n",
+    "    t0 = (time.time() - t0) / iterations\n",
+    "    results_flat = search_redis(redis_client, 'modern art in Europe', k=10, print_results=True)\n",
+    "    print(f\"Flat index query time: {round(t0, 3)} seconds\\n\")\n",
+    "    time.sleep(1)\n",
+    "    print(\" ----- HNSW Index ------ \")\n",
+    "    t1 = time.time()\n",
+    "    for i in range(iterations):\n",
+    "        results_hnsw = search_redis(redis_client, 'modern art in Europe', index_name=HNSW_INDEX_NAME, k=10, print_results=False)\n",
+    "    t1 = (time.time() - t1) / iterations\n",
+    "    results_hnsw = search_redis(redis_client, 'modern art in Europe', index_name=HNSW_INDEX_NAME, k=10, print_results=True)\n",
+    "    print(f\"HNSW index query time: {round(t1, 3)} seconds\")\n",
+    "    print(\" ------------------------ \")\n",
+    "time_queries()"
+   ]
```

Comment:
Consider adding a "call to action / conclusion" at the end with 2-3 bullets.

something like:
1. See a few more example apps of this in action here, here, and here...
2. Have an idea for an OSS app to build with OpenAI and Redis as a vector db? Hit us up here...